pypeit/traceimage.py

```diff
@@ -38,4 +38,9 @@ def __init__(self, spectrograph, files=None, det=1, par=None, bias=None):
         self.process_steps += ['trim']
         self.process_steps += ['apply_gain']
         self.process_steps += ['orient']
+        # TODO: CR masking for the trace images causes major issues with
```

Comment:
Again, I think this is because our combine algorithm is not catching cosmics. Probably an issue with the rejection.

Reply:
probably